firestore/client.go

```diff
@@ -98,12 +101,28 @@ func NewClient(ctx context.Context, projectID string, opts ...option.ClientOptio
 	c := &Client{
 		c:            vc,
 		projectID:    projectID,
-		databaseID:   "(default)", // always "(default)", for now
+		databaseID:   DefaultDatabaseID,
 		readSettings: &readSettings{},
 	}
 	return c, nil
 }
 
+// NewClientWithDatabase creates a new Firestore client that accesses the
+// specified database.
+func NewClientWithDatabase(ctx context.Context, projectID string, databaseID string, opts ...option.ClientOption) (*Client, error) {
+	if databaseID == "" {
+		return nil, errors.New("firestore: databaseID cannot be empty")
```

Comment:
maybe instead, we can write "to create a client using the (default) database, please use `NewClient`"

Reply:
Done